nrfs/include/internal/requests/nrfs_reqs_common.h

```diff
@@ -61,7 +61,8 @@ enum {
 	NRFS_SERVICE_ID_PMIC,
 	NRFS_SERVICE_ID_RESET,
 	NRFS_SERVICE_ID_TEMP,
-	NRFS_SERVICE_ID_USB
+	NRFS_SERVICE_ID_USB,
```

Comment:
This is just adding to the list here but also here a lack of explicit values makes it easy to break the interface if one isn't careful, like trying to keep it alphabetical...